bundles/org.openhab.binding.miio/src/main/resources/database/dreame.vacuum.p2029-miot.json

```diff
@@ -0,0 +1,1076 @@
+					"maximum": 150,
+					"step": 1,
+					"pattern": "%.0f %unit%",
+					"readOnly": true
+				},
+				"refresh": true,
+				"actions": []
+			},
+			{
+				"property": "work-mode",
+				"siid": 4,
+				"piid": 1,
+				"friendlyName": "Vacuum Extend - Work Mode",
+				"channel": "work_mode",
+				"type": "Number",
+				"stateDescription": {
+					"minimum": 0,
+					"maximum": 50,
+					"step": 1,
+					"pattern": "%.0f",
+					"readOnly": true
+				},
+				"refresh": true,
+				"actions": []
+			},
+			{
+				"property": "cleaning-time",
+				"siid": 4,
+				"piid": 2,
+				"friendlyName": "Vacuum Extend - Cleaning Time",
+				"channel": "cleaning_time",
+				"type": "Number:Time",
+				"unit": "minutes",
+				"stateDescription": {
+					"minimum": 0,
+					"maximum": 32767,
+					"step": 1,
+					"pattern": "%.0f %unit%",
+					"readOnly": true
+				},
+				"refresh": true,
+				"actions": []
+			},
+			{
+				"property": "cleaning-area",
+				"siid": 4,
+				"piid": 3,
+				"friendlyName": "Vacuum Extend - Cleaning Area",
+				"channel": "cleaning_area",
+				"type": "Number",
+				"stateDescription": {
+					"minimum": 0,
+					"maximum": 32767,
+					"step": 1,
+					"pattern": "%.0f",
+					"readOnly": true
+				},
+				"refresh": true,
+				"actions": []
+			},
+			{
+				"property": "cleaning-mode",
+				"siid": 4,
+				"piid": 4,
+				"friendlyName": "Vacuum Extend - Cleaning Mode",
+				"channel": "cleaning_mode",
+				"type": "Number",
+				"stateDescription": {
+					"options": [
+						{
+							"value": "0",
+							"label": ""
+						},
+						{
+							"value": "1",
+							"label": ""
+						},
+						{
+							"value": "2",
+							"label": ""
+						},
+						{
+							"value": "3",
+							"label": ""
+						}
+					]
+				},
+				"refresh": true,
+				"actions": [
+					{
+						"command": "set_properties",
+						"parameterType": "STRING"
+					}
+				],
+				"readmeComment": "Value mapping [\"0\"\u003d\"\",\"1\"\u003d\"\",\"2\"\u003d\"\",\"3\"\u003d\"\"]"
+			},
+			{
+				"property": "mop-mode",
+				"siid": 4,
+				"piid": 5,
+				"friendlyName": "Vacuum Extend - Mop Mode",
+				"channel": "mop_mode",
+				"type": "Number",
+				"stateDescription": {
+					"options": [
+						{
+							"value": "1",
+							"label": "Low"
+						},
+						{
+							"value": "2",
+							"label": "Medium"
+						},
+						{
+							"value": "3",
+							"label": "High"
+						}
+					]
+				},
+				"refresh": true,
+				"actions": [
+					{
+						"command": "set_properties",
+						"parameterType": "STRING"
+					}
+				],
+				"readmeComment": "Value mapping [\"1\"\u003d\"Low\",\"2\"\u003d\"Medium\",\"3\"\u003d\"High\"]"
+			},
+			{
+				"property": "waterbox-status",
+				"siid": 4,
+				"piid": 6,
+				"friendlyName": "Vacuum Extend - Waterbox Status",
+				"channel": "waterbox_status",
+				"type": "Number",
+				"stateDescription": {
+					"readOnly": true,
+					"options": [
+						{
+							"value": "0",
+							"label": ""
```

Comment:
Would be good to figure out what is the actual meaning of the value. (e.g. look into the mihome app to see if you can relate this to something you see)
If that is not clear please make the label match the value so that at least it is not a bunch of empty entries that you see.